pytest.ini

```diff
@@ -0,0 +1,70 @@
+[pytest]
+addopts =
+    # `pytest-xdist`:
+    # -n auto  # we need to uncomment it once it's fixed
+    # ^ this needs pytest-xdist to be actually installed
+
+    # show 10 slowest invocations:
+    --durations=10
+
+    # a bit of verbosity doesn't hurt:
+    -v
+
+    # report all the things == -rxXs:
+    -ra
+
+    # tell pytest not to capture stdout
+    -s
+
+    # show values of the local vars in errors:
+    --showlocals
+
+    # autocollect and invoke the doctests from all modules:
+    --doctest-modules
+
+    # dump the test results in junit format:
+    --junitxml=.test-results/pytest/results.xml
+
+    # Fail on non-existing markers:
+    --strict-markers
+
+    # `pytest-cov`:
+    --cov=trustme
```

Comment:
it's best to use `pytest --cov` and then configure coverage in `pyproject.toml` with 
```
source_pkgs = ["trustme"]
source = ["tests"]
```

Reply:
Why? All the things here describe the same tool/setup.

Reply:
`--cov=trustme` adds the source directory, but you want to check tbe coverage of the source package

Reply:
https://github.com/python-trio/trustme/blob/master/ci.sh#L21 this was working around a lack of the `coverage --source-pkgs=` flag